node-launchpad/src/components/status.rs

```diff
@@ -1080,6 +1092,7 @@ impl<T> StatefulTable<T> {
             None => self.last_selected.unwrap_or(0),
         };
         self.state.select(Some(i));
+        self.last_selected = Some(i);
```

Comment:
I think the `TableState` now has a `self.state.select_next(), select_previous(), etc`  fn, I remembered seeing that on a reddit release post last week! Checked it out and it indeed it has one. This can simplify a lot of code here